.travis.yml

```
@@ -19,14 +19,32 @@ env:
         - COASTLINEDIR="$HOME/gmt-install-dir/coast"
         - PATH="$INSTALLDIR/bin:$PATH"
         - LD_LIBRARY_PATH="$INSTALLDIR/lib:$LD_LIBRARY_PATH"
+        - TRAVIS_EVENT_TYPE=cron
+        - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+            export PYTHON=3.6
+            export CONDA_REQUIREMENTS="ci/requirements.txt"
+            export BUILD_DOCS=true
+          fi
 
 before_install:
     # Install GMT dependencies
     - bash ci/travis-setup.sh
+    - if [ "$BUILD_DOCS" == "true" ]; then
```

```suggestion
    - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
```

Could probably merge this block with the one further up. The `env` block is for setting the build matrix.